docker_images/sklearn/tests/test_api.py

```diff
@@ -7,98 +7,143 @@
 # Must contain at least one example of each implemented pipeline
 TESTABLE_MODELS = {
     "tabular-classification": [
-        "skops-tests/iris-sklearn-1.0-logistic_regression-with-config",
-        "skops-tests/iris-sklearn-1.0-logistic_regression-without-config",
-        "skops-tests/iris-sklearn-1.0-hist_gradient_boosting-with-config",
-        "skops-tests/iris-sklearn-1.0-hist_gradient_boosting-without-config",
-        "skops-tests/iris-sklearn-latest-logistic_regression-with-config",
-        "skops-tests/iris-sklearn-latest-logistic_regression-without-config",
-        "skops-tests/iris-sklearn-latest-hist_gradient_boosting-with-config",
-        "skops-tests/iris-sklearn-latest-hist_gradient_boosting-without-config",
+        "skops-tests/iris-sklearn-1.0-logistic_regression-with-config-skops",
```

Comment:
I added a skops test case for every pickle test case **with config**, do you think I should've reduced it? The GHActions never works on these tests anyway, so there's not much limitation except for local dev. @BenjaminBossan @adrinjalali

Reply:
It would be nice if we could eventually get the GH to work, in which case it might matter. Otherwise, I don't have any opinion on that.